sync_diff_inspector/diff.go

```diff
@@ -420,7 +429,7 @@ func (df *Diff) Equal() (err error) {
 
 			if err != nil {
 				log.Error("check failed", zap.String("table", dbutil.TableName(table.Schema, table.Table)), zap.Error(err))
-				return errors.Trace(err)
+				continue
```

Comment:
shoud save the check result like `df.report.FailedNum++`, fix in 
https://github.com/pingcap/tidb-tools/pull/366